examples/material_gallery/lib/gallery/home.dart

```diff
@@ -49,109 +49,81 @@ class GalleryHome extends StatefulWidget {
-                  new GalleryDemo(title: 'Fitness', builder: () => new FitnessDemo())
-                ]
-              ),
-              new GallerySection(
-                title: 'Style',
-                image: 'assets/section_style.png',
-                colors: Colors.green,
-                demos: <GalleryDemo>[
-                  new GalleryDemo(title: 'Colors', builder: () => new ColorsDemo()),
-                  new GalleryDemo(title: 'Typography', builder: () => new TypographyDemo())
-                ]
-              )
+              new GalleryItem(title: 'App Bar', builder: () => new FlexibleSpaceDemo()),
+              new GalleryItem(title: 'Buttons', builder: () => new ButtonsDemo()),
+              new GalleryItem(title: 'Buttons: Floating Action Button', builder: () => new TabsFabDemo()),
+              new GalleryItem(title: 'Cards', builder: () => new CardsDemo()),
+              new GalleryItem(title: 'Chips', builder: () => new ChipDemo()),
+              new GalleryItem(title: 'Date Picker', builder: () => new DatePickerDemo()),
+              new GalleryItem(title: 'Dialog', builder: () => new DialogDemo()),
+              new GalleryItem(title: 'Dropdown Button', builder: () => new DropDownDemo()),
+              new GalleryItem(title: 'Expand/Collapse List Control', builder: () => new TwoLevelListDemo()),
+              new GalleryItem(title: 'Grid', builder: () => new GridListDemo()),
+              new GalleryItem(title: 'Icons', builder: () => new IconsDemo()),
+              new GalleryItem(title: 'Leave-behind List Items', builder: () => new LeaveBehindDemo()),
+              new GalleryItem(title: 'List', builder: () => new ListDemo()),
+              new GalleryItem(title: 'Modal Bottom Sheet', builder: () => new ModalBottomSheetDemo()),
+              new GalleryItem(title: 'Menus', builder: () => new MenuDemo()),
+              new GalleryItem(title: 'Page Selector', builder: () => new PageSelectorDemo()),
+              new GalleryItem(title: 'Persistent Bottom Sheet', builder: () => new PersistentBottomSheetDemo()),
+              new GalleryItem(title: 'Progress Indicators', builder: () => new ProgressIndicatorDemo()),
+              new GalleryItem(title: 'Scrollable Tabs', builder: () => new ScrollableTabsDemo()),
+              new GalleryItem(title: 'Selection Controls', builder: () => new ToggleControlsDemo()),
+              new GalleryItem(title: 'Sliders', builder: () => new SliderDemo()),
+              new GalleryItem(title: 'SnackBar', builder: () => new SnackBarDemo()),
+              new GalleryItem(title: 'Tabs', builder: () => new TabsDemo()),
+              new GalleryItem(title: 'Text Fields', builder: () => new TextFieldDemo()),
+              new GalleryItem(title: 'Time Picker', builder: () => new TimePickerDemo()),
+              new GalleryItem(title: 'Tooltips', builder: () => new TooltipDemo()),
             ]
           ),
-          new Row(
+          new TwoLevelSublist(
```

Comment:
I think we should get rid of this section and its scrolling techniques demo. The real appbar demo is in the components list anyway.